apps/openmw/mwrender/localmap.cpp

```diff
@@ -201,6 +202,13 @@ osg::ref_ptr<osg::Camera> LocalMap::createOrthographicCamera(float x, float y, f
 
     lightSource->setStateSetModes(*stateset, osg::StateAttribute::ON|osg::StateAttribute::OVERRIDE);
 
+    osg::ref_ptr<osg::Image> fakeShadowMapImage = new osg::Image();
+    fakeShadowMapImage->allocateImage(1, 1, 1, GL_DEPTH_COMPONENT, GL_FLOAT);
+    *(float*)fakeShadowMapImage->data() = -std::numeric_limits<float>::infinity(); // this has been tried both positive and negative
+    osg::ref_ptr<osg::Texture> fakeShadowMapTexture = new osg::Texture2D(fakeShadowMapImage);
+    for (int i = SceneUtil::MWShadow::baseShadowTextureUnit; i < SceneUtil::MWShadow::baseShadowTextureUnit + SceneUtil::MWShadow::numberOfShadowMapsPerLight; ++i)
+        stateset->setTextureAttributeAndModes(i, fakeShadowMapTexture, osg::StateAttribute::ON | osg::StateAttribute::OVERRIDE | osg::StateAttribute::PROTECTED);
+
```

Comment:
*Copied from another comment I made because it got erroneously marked as applying to an outdated diff because part of it changed*

Any idea why this *doesn't* work? I think I'm binding a texture to be used as all the shadow maps which has a single pixel with an infinite float value, which should therefore always show as further than the current fragment during the shadow test.